be/CMakeLists.txt

```diff
@@ -79,6 +79,12 @@ option(WITH_BENCH "Build binary with bench" OFF)
 
 option(USE_STAROS "Use StarOS to manager tablet info" OFF)
 
+option(USE_SSE4_2 "Build with SSE4.2 instruction" OFF)
+
+option(USE_AVX2 "Build with AVX2 instruction" OFF)
```

Comment:
I think we'd better make these options' default values be ON

Reply:
ok. but we keep avx512 off by default, right?